scss/variables/colors.js

```diff
@@ -8,41 +8,64 @@
 	}
 }( this, function( chassis ) {
 
-var colors = {
-	"chassis-gray-dark": {
-		"value": "#383838",
-		"name": "Chassis Gray - Dark"
+chassis.colors = {
+	"primary": {
+		name: "Primary Colors",
+		value: {
+			"base": "#3F51B5",
+			"light": "#5C6BC0",
+			"dark": "#3949AB",
+			"darker": "#303F9F"
+		}
 	},
-	"chassis-gray": {
-		"value": "#c1c1c1",
-		"name": "Chassis Gray"
+	"success": {
+			name: "Success Colors",
+			value: {
+				"base": "#8BC34A",
+				"light": "#9CCC65",
+				"dark": "#8BC34A",
+				"darker": "#689F38"
+			}
+		},
+		"warning": {
+			name: "Warning Colors",
+			value: {
+				"base": "#FF9800",
+				"light": "#FFA726",
+				"dark": "#FB8C00",
+				"darker": "#F57C00"
+		}
 	},
-	"chassis-gray-light": {
-		"value": "#f2f2f2",
-		"name": "Chassis Gray - Light"
+		"info": {
+			name: "Info Colors",
+			value: {
+				"base": "#00BCD4",
+				"light": "#26C6DA",
+				"dark": "#00ACC1",
+				"darker": "#0097A7"
+			}
 	},
-	"chassis-blue-dark": {
-		"value": "#388695",
-		"name": "Chassis Blue - Dark"
+	"error": {
+			name: "Error Colors",
+			value: {
+				"base": "#F44336",
+				"light": "#EF5350",
+				"dark": "#E53935",
+				"darker": "#D32F2F"
+			}
 	},
-	"chassis-blue": {
-		"value": "#4fc0c8",
-		"name": "Chassis Blue"
+	"text": {
+			name: "Text Colors",
+			value: {
+				"base": "#212121",
+				"light": "#727272"
+			}
```

Comment:
fails WCAG AAA for normal size text.
Something like `#555555` would pass all tests